src/models/chatglm2.cpp

```diff
@@ -18,10 +18,10 @@
 #include "INIReader.h"
 #include "chatglm2.h"
 
-template <typename WeiT, typename NormT>
-ChatGLM2<WeiT, NormT>::ChatGLM2(const std::string &modelPath, const std::string &modelType)
-    : CommonDecoder<Attention<WeiT, ChatGLM2RotaryEmbedding, NormT, float, float, float, true>,
-            ChatGLM2MLP<WeiT, float, float, float, NormT, true>>(modelPath, modelType) {
+template <typename WeiT>
```

Comment:
previous code contains 2 template parameters:
"template <typename WeiT, typename NormT>"
So, here the normalization operator will always be RmsNorm?

Reply:
Yes, synced w/ Gui Sheng. He answered me it is rms_norm currently in chatglm2/3. 
By the way, ICX could compile template template default param.